app/src/main/res/drawable/ic_contact_mail.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
WARNING: [UnusedResources] The resource `R.drawable.ic_contact_mail` appears to be unused
 Unused resources make applications larger and slow down builds.